python_modules/dagster/dagster/_core/pipes/client.py

```diff
@@ -73,6 +73,10 @@ def inject_context(self, context_data: "PipesContextData") -> Iterator[PipesPara
             process to locate and load the injected context data.
         """
 
+    @abstractmethod
+    def no_messages_debug_text(self) -> str:
+        """A message to be displayed when no messages are received from the external process to aid with debugging."""
+
```

Comment:
An example string would be helpful in communicating the value and intended use case as well as some code as to how that string would be used